content/community/videos.md

```diff
@@ -7,168 +7,46 @@ permalink: community/videos.html
 
-<iframe title="Tom Occhino and Jordan Walke introduce React at Facebook Seattle" width="650" height="366" src="https://www.youtube-nocookie.com/embed/XxVg_s8xAms" frameborder="0" allowfullscreen></iframe>
-
-### Introducing React Native
-
-[Tom Occhino](https://twitter.com/tomocchino) reviews the past and present of React in 2015, and teases where it's going next.
-
-<iframe title="React.js Conf 2015 Keynote - Introducing React Native" width="650" height="366" src="https://www.youtube-nocookie.com/embed/KVZ-P-ZI6W4?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
-
-### Rethinking Web App Development at Facebook
+Lin Clark provides an explanation of what React Fiber is and what it does at React Conf 2017 - (2017 - 0h30min).
+<iframe title="A Cartoon Intro to React Fiber" width="650" height="366" src="https://www.youtube-nocookie.com/embed/ZCuYPiUIONs" frameborder="0" allowfullscreen></iframe>
 
-Delivering reliable, high-performance web experiences at Facebook's scale has required us to challenge some long-held assumptions about software development. Watch this Facebook F8 2014 talk to learn how we abandoned the traditional MVC paradigm in favor of a more functional application architecture.
+A playlist of videos from React.js Conf 2016.
+<iframe title="React.js Conf 2016" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS0M8Q95RIc2lOM6nc77q1IY" frameborder="0" allowfullscreen></iframe>
 
-<iframe title="Hacker Way: Rethinking Web App Development at Facebook" width="650" height="366" src="https://www.youtube-nocookie.com/embed/nYkdrAPrdcw" frameborder="0" allowfullscreen></iframe>
-
-### Secrets of the Virtual DOM
-
-[Pete Hunt](http://www.petehunt.net/) at Mountain West JavaScript 2014 discusses why a virtual DOM was built for React, how it compares to other systems, and its relevance to the future of browser technologies.
+A playlist of videos from React.js Conf 2015.
+<iframe title="React.js Conf 2015" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
 
+Pete Hunt at Mountain West JavaScript 2014 discusses why a virtual DOM was built for React, how it compares to other systems, and its relevance to the future of browser technologies - (2014 - 0h44m).
 <iframe title="Mountain West JavaScript 2014 - Be Predictable, Not Correct. by Pete Hunt" width="650" height="366" src="https://www.youtube-nocookie.com/embed/h3KksH8gfcQ" frameborder="0" allowfullscreen></iframe>
 
-### Rethinking Best Practices
+How Facebook abandoned the traditional MVC paradigm in favor of a more functional application architecture - (2014 - 0h44m).
+<iframe title="Hacker Way: Rethinking Web App Development at Facebook" width="650" height="366" src="https://www.youtube-nocookie.com/embed/nYkdrAPrdcw" frameborder="0" allowfullscreen></iframe>
```

Comment:
This same video (www.youtube-nocookie.com/embed/nYkdrAPrdcw) appears to be duplicated 3x on the new page.

Reply:
Yikes! I must have copied the same iframe to multiple places when I was replacing the links with the previous iframes. I'll change those to link to the correct video and recommit! 😄